stable/traefik/templates/acme-pvc.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.acme.enabled .Values.acme.persistence.enabled (not .Values.acme.persistence.existingClaim) }}
+{{- if and .Values.acme.enabled .Values.acme.persistence.enabled (not .Values.acme.persistence.existingClaim) (not .Values.consul.enabled) }}
```

Comment:
I understand what you're trying to do in this PR, but there's something that feels very unintuitive about the idea that you enabled consul and the result is that you _don't_ get a PVC _even if_ persistence is set to enabled.

I don't have a specific suggestion about how to mitigate that, but I think it's something you should consider.

Reply:
You're right, it might be confusing... Also, having a Consul backend doesn't necessarily mean you want to store the ACME certs in Consul. I'll think on this and maybe split it out so it's all a bit more specific.

Reply:
Should we not add a note in Readme that Persistence won't be enabled if `.Values.consul.enabled`?

Reply:
@veeramarni I'm gonna look into doing all of this in a more graceful way, personal time lacking a bit to work on this right now.

Reply:
@siwyd Just checking if you get a chance to improve it?